src/background/wallet.js

```diff
@@ -70,8 +72,8 @@ export async function init() {
   connectionsQueue.forEach(addAeppConnection);
   connectionsQueue = [];
 
```

Comment:
The network change is no longer being propagated to the connected aepp. It is working in `feat/multichain`.

Reply:
Fixed. Can you please test this? I also informed @Liubov-crypto about this issue and she will retest it.